packages/terra-core-docs/src/terra-dev-site/doc/profile-image/AccessibilityGuide.4.doc.mdx

```diff
@@ -0,0 +1,119 @@
+import { Notice } from '@cerner/terra-docs';
+import ProfileImageExamples from './example/ProfileImageExamples';
+
+<Badge />
+
+# Why the accessibility of Profile Image is important 
+
+<Notice variant="important" ariaLevel="2">
+
+The Profile Image component represents an account or profile across the platform. A Profile Image visually associates a user’s, patient’s, or customer’s image with personalised content.
+
+Poor or improper implementation can cause Profile Image to be inaccessible, which can cause user confusion and impede the identification of content relevant to that user’s profile.
+
+</Notice>
+
+## Accessibility Considerations
+
+### Content Considerations: What do content creators need to do?
+
+> - Content creators must ensure that images with meaningful content have a valid and a textual equivalent.
+>   - Provide engineers with text for the “alt prop”.
+>   - The alt prop must be a textual equivalent that conveys the meaning of the image.
+> - For supplemental or decorative images that provide no contextual value, content creators must convey to the engineers that no alt prop will be supplied.
+> - Content creators must work with engineers to convey any visual relationships created with Profile Image and other content on the page.
+
+### Code Considerations: What do engineers need to do?
+
+> - Engineers must not override the Profile Image’s role of image.
+> - Engineers must apply an alt prop for meaningful images.
+<div aria-label="Example Demo">
+  <ProfileImageExamples />
+</div>
+> ```jsx
+> const cx = classNames.bind(styles);
+> 
+> const ProfileImageExamples = () => (
+>   <div>
+>     <Card>
+>       <Card.Body>
+>         <ProfileImage alt="" width="75" height="75" aria-describedby="caption-example-1" />
+>         <span id="caption-example-1" className={cx('add-left-margin')}>Decorative Profile Image Source</span>
+>       </Card.Body>
+>     </Card>
+>     <Card>
+>       <Card.Body>
+>         <ProfileImage alt="profile image for Martin Moon, photograph from space of rising star over a planet's silhouette" src={exampleProfileImage} aria-describedby="caption-example-2" />
+>         <span id="caption-example-2" className={cx('add-left-margin')}>Meaningful Profile Image Source</span>
+>       </Card.Body>
+>     </Card>
+>   </div>
+);
+> ```
+> - Work with content creators to understand how content should resize and reflow at different breakpoints and form factors to ensure content is not artificially constrained or cut off at smaller viewports.
+> - Engineers must work with the content creators to understand if Profile Image is visual related to other content.
+>   - If so, the engineer must associate the Profile Image to the related content.
+
+## Usability Expectations
+
+### Interaction Detail
+
+Profile image is not interactable.
+
+### Usability Expectations
+
+The user expects to understand the purpose of Profile Image regardless of how they experience the content.
```

Comment:
Instead of having the usability expectations heading twice this section should be moved under the first heading iteself.